phone_number.rb

```diff
@@ -0,0 +1,63 @@
+require_relative "./phone_actions"
+
+class PhoneNumber
+  USERNAME = ENV['ACTIONS_USERNAME']
+  PASSWORD = ENV['ACTIONS_PASSWORD']
+
+  def initialize(number)
+    @number = number
+    @number = format_number
```

Comment:
`@number` is reassigned twice.  It looks like you want to store both representations of the phone number.  Would this work better for you?
```suggestion
    @formatted_number = format_number
```